cli/src/find_path.rs

```diff
@@ -0,0 +1,62 @@
+use super::pager::Pager;
+use clap::ArgMatches;
+use error::Result;
+use fmt::{Formatter, Simple};
+use stellar_client::{sync, endpoint::payment, resources::Amount, resources::AssetIdentifier,
+                     sync::Client};
+
+pub fn find_path(client: &Client, matches: &ArgMatches) -> Result<()> {
+    let pager = Pager::from_arg(&matches);
+
+    let endpoint = {
+        let destination_account = matches
+            .value_of("destination_account")
+            .expect("Destination account is a required field");
+        let source_account = matches
+            .value_of("source_account")
+            .expect("Source account is a required field");
+        let destination_asset_type = matches
+            .value_of("destination_asset_type")
+            .expect("Destination asset type is a required field");
+        let destination_amount = matches
+            .value_of("destination_amount")
+            .expect("Destination amount is a required field");
+        let destination_asset_issuer = matches
+            .value_of("destination_asset_issuer")
+            .map(|issuer| issuer.to_string());
+        let destination_asset_code = matches
+            .value_of("destination_asset_code")
+            .map(|code| code.to_string());
+
+        let destination_asset = AssetIdentifier::new(
+            destination_asset_type,
+            destination_asset_code,
+            destination_asset_issuer,
+        )?;
+
+        let endpoint = payment::FindPath::new(
+            source_account,
+            destination_account,
+            destination_asset,
+            Amount::new(destination_amount.parse::<i64>().unwrap() * 10000000),
```

Comment:
I can understand why this doesn't make you feel good.  We ultimately decided that because stellar works with stroops and displays units scaled down by 10^7 to its users, we would follow the same convention. https://www.stellar.org/developers/guides/concepts/assets.html#amount-precision-and-representation . 

I would recommend adding underscores to that big number where a comma would typically go i.e. `10_000_000`